tests/js/realm-tests.js

```diff
@@ -117,6 +117,16 @@ module.exports = BaseTest.extend({
         TestCase.assertEqual(Realm.defaultPath, newPath, "defaultPath should have been updated");
     },
 
+    testRealmSchemaVersion: function() {
+        TestCase.assertEqual(Realm.schemaVersion(Realm.defaultPath), 0xFFFFFFFFFFFFFFFF);
```

Comment:
I think this should either return `-1` or throw. I think my vote is on the former.